zuul/tenants.yaml

```diff
@@ -100,6 +100,7 @@
               - ansible/galaxy
               - ansible/pylibssh
               - ansible/workshops
+              - ansible/galaxy-api
```

Comment:
you should add this to https://github.com/ansible/project-config/pull/424

Reply:
done.